Month1/Week2/Section1/Exercise3/Main3.cpp

```diff
@@ -150,18 +135,16 @@ int main()
 	std::cout << "Your choice: " << Choice << std::endl;
 	if (Choice == 1) 
 	{
-		Circle circle;
-		circle.DataCircle();
+		FCircle circle;
+		circle.InPutDataCircle();
 		circle.GetAreaCicle();
 		circle.GetPerimeterCircle();
-		circle.PrinterCircle();
 	}
 	if (Choice == 2)
 	{
-		Square square;
-		square.DataSquare();
+		FSquare square;
```

Comment:
Same thing here. `square` does not follow convention.

Reply:
changed